interview-questions.md

```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
-Your answer:
+Your answer: git is a software that was created to allow developers to share code freely between eachother. Github is a service that acts as an interface for git, making it easier to use.
 
-Researched answer:
+Researched answer: Git is a version control software that was originally designed to make coordination in coding tasks easier, and has since been used moreso to keep track of changes made to codebases or files in general. Github is a git repository, based on the web and using cloud storage, that acts as an easier way to use git and adds its own user features that make git collaboration better for coders.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: The Boolean operators return boolean values. These are the operators like and, or, and not. They are often used to check if a statement is true or false within if/else statements.
 
-Researched answer:
+Researched answer: Boolean operators are also very useful for creating complex logic within code, and are a sort of building block in that sense. They can be used to create complicated decision trees and are present in almost every form of logical code.
 
 3. What is an index? What is the difference between index and value?
 
-Your answer:
+Your answer: An index is a position in a structure, for example an array. Because javascript uses a zero index, the thing stored in the first position will be index 0, the next will be 1, and so on. A value is a piece of information that is stored to be used, and can be anything, without an order.
 
-Researched answer:
+Researched answer: An index is a type of value, specifically used to keep track of positions of data within a collection. This means that when using any data structure that has an ordered set of values, you are innately using an index for each. A value can be a wider range of data, including built-in uses like the index of an array, or the data input and stored by a user.
 
 4. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are different.
 
-Your answer:
+Your answer: Arrays and strings are similar in that they hold an ordered series of values. In strings, these values are always a single character. In arrays these values can be many data types, including strings themselves, so they are more commonly used for storage of other values while a string is often itself the data you want stored.
 
-Researched answer:
+Researched answer: The main diiference between Arrays and Strings are that one, the string, is a data type that is comprised of characters, and an array is a data structure used to hold a collection of data. An array can hold many elements of the same type, though they are not limited to a single form of data like strings. They are similar in that they can both be manipulated in similar ways, and strings can even be easily represented by an array of characters.
 
```

Comment:
Love this researched answer! I want to encourage you to use examples and expand on the idea that they can both be manipulated in similar ways. Which methods can we use on both for example.